services/imageService.js

```diff
@@ -0,0 +1,36 @@
+const DatauriParser = require('datauri/parser')
+const userModel = require('../models/users')
+const { upload, generateUrl } = require('../utils/cloudinary')
+const cloudinaryMetaData = require('../constants/cloudinary')
+
+/**
+ * upload user profile picture to cloudinary
+ *
+ * @param file { Object }: multipart file data
+ * @param userId { string }: User id
+ */
+const uploadProfilePicture = async (file, userId) => {
+  try {
+    const parser = new DatauriParser()
+    const imageDataUri = parser.format(file.originalname, file.buffer)
+    const imageDataInBase64 = imageDataUri.content
+    const uploadResponse = await upload(imageDataInBase64, {
+      folder: cloudinaryMetaData.PROFILE.FOLDER,
+      tags: cloudinaryMetaData.PROFILE.TAGS
+    })
```

Comment:
We're uploading the file the way it is, we're not resizing it, am I right? 
If yes, is that in the future scope of this feature?

Reply:
We won't be making changes to the original image,
Url can be configured and image can be resized dynamically as per requirement,